data/base/script/campaign/libcampaign_includes/artifact.js

```diff
@@ -150,6 +150,12 @@ function __camPickupArtifact(artifact)
 		camTrace("Artifact", artifact.id, "is not managed");
 		return;
 	}
+	if (Object.hasOwn(ai, "pickedUp") && ai.pickedUp === true)
+	{
+		camDebug("Already picked up the artifact", __ALABEL);
```

Comment:
Should possibly be:
```suggestion
		camTrace("Already picked up the artifact", __ALABEL);
```
To avoid spamming console if debug mode is disabled? (@KJeff01 ?)

Reply:
This is probably a very rare event, but yes, `camTrace()` would be better here. `camTrace()` only outputs notifications while cheat mode is active, while `camDebug()` should be used for actual bugs (scenarios that shouldn't happen).

Reply:
Got it. Will fix.

Reply:
Done